lib/string_basics.dart

```diff
@@ -3,9 +3,112 @@
 // license that can be found in the LICENSE file.
 
 import 'src/slice_indices.dart';
+import 'package:quiver/strings.dart' as quiver_strings;
 
 /// Utility extension methods for the native [String] class.
 extension StringBasics on String {
+  /// Returns a value according to the contract for [Comparator] indicating
+  /// the ordering between [this] and [other], ignoring letter case.
+  ///
+  /// Example:
+  /// ```dart
+  /// 'ABC'.compareTo('abd'); // negative value
+  /// 'ABC'.compareTo('abc'); // zero
+  /// 'ABC'.compareTo('abb'); // positive value
+  /// ```
```

Comment:
Can you add an empty comment line after this? To match the formatting everywhere else

Reply:
Done.